pki.spec

```diff
@@ -33,41 +33,16 @@ Source: https://github.com/dogtagpki/pki/archive/v%{version}%{?_phase}/pki-%{ver
 # NSS
 ################################################################################
 
-%if 0%{?rhel} && 0%{?rhel} <= 7 || 0%{?fedora} && 0%{?fedora} <= 27
-%global nss_default_db_type dbm
-%else
 %global nss_default_db_type sql
-%endif
 
 ################################################################################
 # Python
 ################################################################################
 
-# Python 2 packages
-%if 0%{!?with_python2:1}
-%if 0%{?rhel} && 0%{?rhel} <= 7 || 0%{?fedora} && 0%{?fedora} <= 28
-%global with_python2 1
-%else
-# no python2
-%endif
-%endif
-
-# Python 3 packages
-%if 0%{!?with_python3:1}
-%if 0%{?rhel} && 0%{?rhel} <= 7
-# no python3
-%else
-%global with_python3 1
-%endif
-%endif
-
-# Use Python 3 for all commands?
-%if 0%{!?with_python3_default:1}
-%if 0%{?rhel} && 0%{?rhel} <= 7 || 0%{?fedora} && 0%{?fedora} <= 27
-%global with_python3_default 0
+%if 0%{?rhel}
```

Comment:
`%if 0%{?rhel} && 0%{?rhel} >= 8`

This isn't present on RHEL 7 (though admittedly, `pki` master isn't expected to build on RHEL 7).

Reply:
Ah never mind. You're dropping unsupported platforms at the same time. This is OK then.

Reply:
Right. PKI 10.8 will only be supported on F30+ and RHEL 8.2+.